### ISSUE TITLE
Fix errors again

### ISSUE THREAD

**Issue by Taberone** [Taberone/Cataclysm-DDA-Touhou-Mod#42]
Despite this mod being on life support, newer experimentals have errors.

![image](https://user-images.githubusercontent.com/9733597/109441150-537d6600-79e9-11eb-901d-27f22f435cb2.png)
![image](https://user-images.githubusercontent.com/9733597/109441916-8c1e3f00-79eb-11eb-9733-02bf1f99ba3b.png)
![image](https://user-images.githubusercontent.com/9733597/109441924-92acb680-79eb-11eb-8982-fafd45034f73.png)
![image](https://user-images.githubusercontent.com/9733597/109441931-98a29780-79eb-11eb-8f0f-0cf0b35d4e8a.png)
![image](https://user-images.githubusercontent.com/9733597/109441941-9cceb500-79eb-11eb-9a74-867ece9abef3.png)
![image](https://user-images.githubusercontent.com/9733597/109441945-a0623c00-79eb-11eb-9582-6479fdfc7870.png)
![image](https://user-images.githubusercontent.com/9733597/109441953-a5bf8680-79eb-11eb-89f5-9565b842f903.png)
![image](https://user-images.githubusercontent.com/9733597/109441973-b40da280-79eb-11eb-9733-bf04d517cbb1.png)

**Comment by Taberone** [comment]
There's a lot more errors than just the screenshots. Goddamn, I'm not sure if I'll even have time for this.

**Comment by miu135** [comment]
Did an update broke it or something?

**Comment by Taberone** [comment]
Experimentals get updated constantly, which also tends to break stuff constantly. I don't play CDDA as much as I used to, unfortunately. Pretty much everything has to be redone to fix the errors or something.

**Comment by Taberone** [comment]
Honestly, not sure if I even have the willpower to fix the mod. I've kinda moved on to other games, and college-related stuff also saps my time. I've kinda burnt out on constantly keeping the mod on life support because the Experimentals keep breaking everything with each new update.

I hate to leave things unfinished, but I think I've overextended myself a bit far with the original sins of this mod. (Not using copy-from until it was too late, overconfidence, overestimating my ability, etc) If I ever get the itch to play CDDA again, I might try to come back to this. But as it stands, the mod is pretty much in a coma.

If you have any modding knowledge by any chance, you're free to fork the mod and make your own changes or versions, I guess.

**Comment by RedMisao** [comment]
Hello Taberone, I tried to make the mod compatible with the latest version of the game (0.F-2), by fixing these errors you were referring to, here are most if not all the changes needed to make it run:

![image](https://user-images.githubusercontent.com/95231551/145737925-3eb16b84-c37d-429e-8a26-62dd3ee9c888.png)

I then realized, why don't I add more stuff to it? This is one of the few, if not the only game that lets you play not only as a gyokuto but as Reisen, the one from your profile pic. I'm in process of adding more professions, rudimentary spells, clothing, items and traits/perks. I have some plans to add each character's species (in some form), plus mobs/monsters NPCs, locations and a story related to Gensokyo
However, there are some differences in how I'd like to implement some stuff. For example the gyokuto you seem to had in mind were cowardly monsters, but I want to use them as highly competent NPC fighters from a new faction (you may already guess which one)
I don't have the base, 0.F-2 compatible .zip anymore, but I can replicate it if needed

**tl;dr** I forked the repo and I'm expanding the mod's contents, using the last stable release for C:DDA (0.F-2) for testing. I'm two or three big number versions away from adding story stuff

**Comment by miu135** [comment]
Is this compatible with BN?

**Comment by RedMisao** [comment]
Probably. I'm running base cdd + Magiclysm, adding stuff only to the starting characters, not worldgen yet.
I have Fuji's military professions + cityside running along too but I haven't touched structure generation so far.

**Comment by Taberone** [comment]
> Hello Taberone, I tried to make the mod compatible with the latest version of the game (0.F-2), by fixing these errors you were referring to, here are most if not all the changes needed to make it run:
> 
> ![image](https://user-images.githubusercontent.com/95231551/145737925-3eb16b84-c37d-429e-8a26-62dd3ee9c888.png)
> 
> I then realized, why don't I add more stuff to it? This is one of the few, if not the only game that lets you play not only as a gyokuto but as Reisen, the one from your profile pic. I'm in process of adding more professions, rudimentary spells, clothing, items and traits/perks. I have some plans to add each character's species (in some form), plus mobs/monsters NPCs, locations and a story related to Gensokyo However, there are some differences in how I'd like to implement some stuff. For example the gyokuto you seem to had in mind were cowardly monsters, but I want to use them as highly competent NPC fighters from a new faction (you may already guess which one) I don't have the base, 0.F-2 compatible .zip anymore, but I can replicate it if needed
> 
> **tl;dr** I forked the repo and I'm expanding the mod's contents, using the last stable release for C:DDA (0.F-2) for testing. I'm two or three big number versions away from adding story stuff

That's nice. Do what you want with making the moon rabbits more competent fighters, I guess. I barely even play CDDA anymore.

**Comment by RedMisao** [comment]
Hello, @Taberone. I finished (a large portion) of the mod and I'm looking forward to PR it to the CDDA mainline repo (here's the link for the PR https://github.com/CleverRaven/Cataclysm-DDA/pull/62930), but I need your github mail to add you as co-author. Could you please post it here?

**Comment by miu135** [comment]
@RedMisao You might have better luck with BN

**Comment by Taberone** [comment]
Whoops, I haven't been on Github in a while and for the longest time kinda just forgot my password. Do you still need it? The thing above seems closed...

I've edited my profile to include my Github Mail thingy, if it helps.
